documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,103 @@
+> Release Issue Template
+
+# Lotus X.Y.Z Release
+
+We're happy to announce Lotus X.Y.Z...
+
+## 🗺 What's left for release
+
+<List of items with PRs and/or Issues to be considered for this release>
+
+## 🚢 Estimated shipping date
+
+<Date this release will ship on if everything goes to plan (week beginning...)>
+
+## 🔦 Highlights
+
+< top highlights for this release notes >
+
+## Changelog
+
+< changelog generated by bin/mkreleaselog >
+
+## ✅ Release Checklist
+
+First steps:
+
+  - [ ] Fork a new branch (`release/vX.Y.Z`) from `master` and make any further release related changes to this branch. If any "non-trivial" changes get added to the release, uncheck all the checkboxes and return to this stage.
+  - [ ] Prep the changelog
+  - [ ] Bump the version in `version.go` in the `master` branch to `vX.(Y+1).0-dev`.
+  - [ ] Follow the RC release process to cut the first RC.
+    
+Prepping an RC:
+
+- [ ] version string in `build/version.go` has been updated (in the `release/vX.Y.Z` branch).
+- [ ] tag commit with `vX.Y.Z-rcN`
+- [ ] cut a pre-release on [github](https://github.com/filecoin-project/lotus/releases)
+
+Testing an RC:
+
+- [ ] **Stage 0 - Automated Testing**
+  - Automated Testing
+    - [ ] CI: Ensure that all tests are passing.
+    - [ ] Testground tests
```

Comment:
Do these automatically run?  If not, where does someone look to confirm this?

Reply:
They do. @coryschwartz what's the status on getting the testground test results back into lotus?

Reply:
I have a couple of open PRs for working with TG. The main one is https://github.com/filecoin-project/lotus/pull/5887, which just adds testground tests at the bottom of PRs. The other PRs are a workflow for running more or longer-running testground tests that get triggered just during release...like on a release issue, or a "chatops" interface in which specific tests can be asked for in issues or PRs and their results included in github discussions. 
```
https://github.com/filecoin-project/lotus/pull/5888
https://github.com/filecoin-project/lotus/pull/5887
https://github.com/filecoin-project/lotus/pull/5916
```